embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -1,36 +1,39 @@
 package io.embrace.android.embracesdk.arch.schema
 
-import io.embrace.android.embracesdk.internal.spans.toEmbraceAttributeName
-
-internal sealed class EmbType {
+internal sealed class EmbType(type: String, subtype: String?) : TelemetryType {
+    override val attributeName: String = "type"
+    override val attributeValue = type + (subtype?.run { ".$this" } ?: "")
 
     /**
      * Keys that track how fast a time interval is. Only applies to spans.
      */
-    internal object Performance : TelemetryType {
-        override val description: String = "performance"
+    internal sealed class Performance(subtype: String?) : EmbType("performance", subtype) {
+
+        internal object Default : Performance(null)
```

Comment:
Does it make sense to include these for other classes in the hierarchy if we're going to include it?